docs/source/debugging.md

```diff
@@ -0,0 +1,67 @@
+# Debugging tips
+
+Sometimes `constructor` fails to build the requested installer. Sometimes the installer builds, but
+there are errors when it is run in the target machine.
+
+In this section we will talk about different techniques you can use to find out was wrong with your installers.
+
+## Run `constructor` in debug mode
+
+When you run `constructor`, `conda` is called behind the scenes to solve the requested specs. You can enable advanced logging using the flags `-v` or `--debug`. These will be passed to `conda`. Be prepared for _a lot_ of output!
+
+
+## Verbose `conda`
+
+Whether it's while running `constructor` to build an installer, or while the installer is running in the target machine, some instance of `conda` will be running behind the scenes. You can request more verbose output via the `CONDA_VERBOSITY` environment variable. It can take values from `1` to `4`. You can also set `CONDA_DEBUG` to `1`.
+
+
+## Verbose shell installers
+
+The shell installers are "just" a shell script that invokes `conda` in certain events. This means you can enable `bash` verbosity via the `-x` flag:
+
+```bash
+$ bash -x ./Miniconda.sh
+# or with verbose conda:
+$ CONDA_VERBOSITY=3 bash -x ./Miniconda.sh
+```
+
+## Verbose PKG installers
+
+PKG installers are usually invoked graphically. You can check the native logs via <kbd>⌘</kbd>+<kbd>L</kbd>. Note that you will need to choose the detailed view in the dropdown menu you'll find in the top right corner.
+
+In order to get more verbosity out of conda, you now know you need the `CONDA_VERBOSITY` variable. However, it needs to be set up _before_ running the installer. One way from the command line would be:
```

Comment:
```suggestion
In order to get more verbosity out of `conda`, you now know you need the `CONDA_VERBOSITY` variable. However, it needs to be set up _before_ running the installer. One way from the command line would be:
```

Typically we only monospace "conda" if it's referred to as a command, but it seems like constructor docs monospace both "`constructor`" and "`conda`" so I'm suggesting this change here for the sake of consistency